firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Ranges.txt

```diff
@@ -0,0 +1,100 @@
+Scenario:The Ranges:  clamp and within function works correctly
+  GIVEN('The clamp and within function')
```

Comment:
@rohanpurohit Last week I went over this file with you and discussed some other possible ways to rework the structure of Ranges (e.g. having the GIVENs be the floor, value, and ceiling, rather than GIVEN clamp and within functions). Would you be able to work with Raavi over Zoom to interactively rework the structure, as well as making other improvements along our overall test pattern (e.g. the WHENs in this file should name which function is being called, which is not happening consistently in this GIVEN)?

Reply:
Yes, @ethanjli I can do that. we'll go over several iterations and maybe you can review once we think it's ready for review from our end.